### ISSUE TITLE
Should subroutine vernalization have some dtime dependence?

### ISSUE THREAD

**Issue by billsacks** [ESCOMP/CTSM#1629]
I noticed this when investigating the possible impacts of #1626 : It looks like the vernalization subroutine in CNPhenologyMod accumulates some variables each time it's called, but there is no multiplication by dtime. I haven't come to fully understand what's going on in that subroutine, but it seems suspicious that there is no dtime multiplier: If this subroutine was called twice as often (due to a decrease in time steps from 30 to 15 min), would vernalization accumulation happen twice as fast?

@slevisconsulting is this something you're familiar with?

**Comment by ekluzek** [comment]
One thing to note, that the vernalization is used for winter wheat, which we aren't doing right now. So I think for now it's not working and so never called. Someone else chime in if I have that wrong though.

**Comment by slevisconsulting** [comment]
@billsacks I don't think that I ever tested vernalization beyond ensuring that it didn't crash. I don't know if anyone else ever looked at it. Maybe Yiqi Lu (postdoc from a few years ago)? From your description, I'm guessing that everything you've said above is correct.

**Comment by billsacks** [comment]
Thanks. I'm going to change the title of this issue and tentatively mark it as a bug.
